x-pack/plugins/canvas/canvas_plugin_src/renderers/external.ts

```diff
@@ -26,4 +27,4 @@ export const renderFunctions = [
   progressRenderer,
 ];
 
-export const renderFunctionFactories = [];
+export const renderFunctionFactories = [debugRendererFactory, errorRendererFactory];
```

Comment:
Is including these renderers here causing them to be double registered, since htey are also being registered in the errorPlugin?